### ISSUE TITLE
Overflow shows absolutely nothing

### ISSUE THREAD

**Issue by Intecpsp** [isabellaalstrom/lovelace-grocy-chores-card#107]
![Screenshot 2023-04-19 at 11 48 48 AM](https://user-images.githubusercontent.com/1164012/233132014-91b80e72-bbdf-44ac-b0ce-b68f259c1667.jpg)
![Screenshot 2023-04-19 at 11 48 57 AM](https://user-images.githubusercontent.com/1164012/233159012-198309e8-3c28-4e8e-ac30-66e3ee15b895.jpg)

I'm not sure when this stopped working, because it did at one point.

Lovelace Grocy Chores Card version: 3.6.0
Home Assistant version: 2023.4.5
Frontend version: 20230411.1
card-tools version: 11

Here is this cards config:

```
title:  chores
type: custom:grocy-chores-card
entity:
 sensor.grocy_chores
 sensor.grocy_tasks
show_quantity: 5
show_chores_without_due: false
show_tasks_without_due: false
filter_user: 3
show_last_tracked_by: false
show_track_button: false
show_overflow: true
show_divider: true
due_in_days_threshold: 7
use_24_hours: false
hide_text_with_no_data: true
```

**Comment by Intecpsp** [comment]
I decided to check out the element and found that they're all there, just not being displayed
![Screenshot 2023-04-19 at 1 29 22 PM](https://user-images.githubusercontent.com/1164012/233154003-fda3e6e3-ad69-4ce9-b7c3-b3dd296e2bdc.jpg)

**Comment by Intecpsp** [comment]
Not sure why this JUST NOW occurred to me, but downgrading to 3.5.1 corrects this. So something in 3.6.0 broke this functionality.

**Comment by karwosts** [comment]
Looks like there was a recent change that added show-class/hidden-class labels on all the individual items, and the toggleOverflow function is now toggling too many things. 

Put a fix in #108
